src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --main-color: #00acdc;
```

Comment:
It's a good practice to define your CSS variables in the `:root` pseudo-class, which represents the root (top-level element) of the document, rather than the `body` or other elements.